code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -122,8 +132,9 @@
 	var/list/possible_targets = ListTargets()
 	if(environment_smash)
 		EscapeConfinement()
-
 	if(AICanContinue(possible_targets))
+		if(!attack_is_on_cooldown)
+			TryAttack()
```

Comment:
Im going to test this with monsters that attack in unconventional ways like saplings to see if theres any weirdness.

Reply:
Found no issues with Wolf, Snow Whites Apple, or Parasite Tree Saplings. Those are the 3 i can think of who have unconvientional attack methods.